plugin/pkg/scheduler/schedulercache/cache.go

```diff
@@ -233,6 +233,44 @@ func (cache *schedulerCache) RemovePod(pod *api.Pod) error {
 	return nil
 }
 
+func (cache *schedulerCache) AddNode(node *api.Node) error {
+	cache.mu.Lock()
+	defer cache.mu.Unlock()
+
+	n, ok := cache.nodes[node.Name]
+	if !ok {
+		n = NewNodeInfo()
+		cache.nodes[node.Name] = n
+	}
+	return n.SetNode(node)
+}
+
+func (cache *schedulerCache) UpdateNode(oldNode, newNode *api.Node) error {
```

Comment:
The Update seems to accomodate the controller model of Add, Update, Delete funcs.
If a node's fields was changed, e.g. status, resources, then it should be called.
Not quite useful in current use cases, I guess.